partitura/io/importkern.py

```diff
@@ -351,7 +368,7 @@ def load_kern(
     )
 
     doc_name = get_document_name(filename)
-    score = spt.Score(partlist=partlist, id=doc_name)
+    score = spt.Score(partlist=partlist[::-1], id=doc_name)
```

Comment:
add a comment stating that we are inversing the order, and why we need to do that

Reply:
I think I had this somewhere.... thank I will add a comment